packages/flutter/lib/src/widgets/app.dart

```diff
@@ -910,6 +915,8 @@ class WidgetsApp extends StatefulWidget {
   /// the [actions] for this app. You may also add to the bindings, or override
   /// specific bindings for a widget subtree, by adding your own [Actions]
   /// widget.
+  ///
+  /// Passing this will not replace [DefaultTextEditingActions].
```

Comment:
For shortcuts, the default text editing shortcuts need to be placed inside the defaultShortcuts because they need able to fall through to the defaultShortcuts when an EditableText isn't focused.  I'll make actions do the same thing and update the docs to match.